microsoft-edge/devtools-guide-chromium/issues/index.md

```diff
@@ -20,118 +20,188 @@ keywords: microsoft edge, web development, f12 tools, devtools
+
+    :::image type="complex" source="../media/issues-tool-initial-view-accessibility-page.msft.png" alt-text="The Issues tool for the Accessibility demo page" lightbox="../media/issues-tool-initial-view-accessibility-page.msft.png":::
+       The **Issues** tool for the Accessibility demo page
+    :::image-end:::
+
+
+## View issues in context of an associated tool
+
+An issue in the Issues tool typically includes one or more links that open a different tool, such as **Elements**, **Network**, or **Sources**.
+
+1.  Select a resource link, to view the resource in the appropriate context within DevTools, such as the **Network** tool or the **Elements** tool.  For example, in the **Issues** tool for the SameSite cookies demo page, select `samesite-sandbox.glitch.me` under **Requests** to show the cookies that are attached to that request.  The cookies are then displayed in the **Network** tool.
+
+    :::image type="complex" source="../media/issues-tab-view-issue.msft.png" alt-text="None value in the SameSite column for the ck02 cookie in the Network tool" lightbox="../media/issues-tab-view-issue.msft.png":::
+       `None` value in the **SameSite** column for the `ck02` cookie in the **Network** tool
+    :::image-end:::
+
+1.  Scroll to view the item with a problem, such as the `ck02` cookie.  In the **SameSite** column, hover over the circled `i` \(![Cookie issue in Network tool](../media/network-tool-cookie-issue-icon.msft.png)\) next to **None**.  The resulting tooltip provides information similar to the **Issues** tool.
+
+    For the accessibility-testing demo page, issues have links to open the **Elements**, **Sources**, or **Network** tool.
+
+    :::image type="complex" source="../media/issues-tool-affected-resource-opens-elements-tool.msft.png" alt-text="Selecting an affected resource in the Issues tool for the accessibility demo page opens the Elements tool" lightbox="../media/issues-tool-affected-resource-opens-elements-tool.msft.png":::
+       Selecting an affected resource in the **Issues** tool for the accessibility demo page opens the **Elements** tool
+    :::image-end:::
+
+
+## Open issues from the DOM tree
+
+If an element has an issue, the DOM tree in the **Elements** tool shows wavy underlining under the element name.  You can either **Shift+click** the wavy-underlined element as described below, or right-click an element that has a wavy underline and then select **View issues**.
+
+To view an issue for an element that has wavy underlining in the DOM tree:
+
+1.  Open a page, such as the [accessibility-testing demo page](https://microsoftedge.github.io/DevToolsSamples/a11y-testing/page-with-errors.html), in a new tab or window.
+
+1.  Select `F12` to open DevTools, and then select the **Elements** tab.
```

Comment:
Do we need to explain opening DevTools here? If so, I'd recommend removing the `F12` part since there are several ways to open DevTools